test/fixers/test_dprint_fixer_callback.vader

```diff
@@ -0,0 +1,28 @@
+Before:
+  call ale#assert#SetUpFixerTest('typescript', 'dprint')
+
+After:
+  Restore
+  call ale#assert#TearDownFixerTest()
+
+Execute(The dprint callback should return the correct default values):
+  AssertFixer
+  \ {
+  \   'read_temporary_file': 1,
+  \   'command':
+  \     ale#Escape('dprint')
+  \     . ' ' . ale#Escape('')
+  \     . ' %t',
+  \ }
+
+Execute(The dprint callback should include custom options):
+  let g:ale_typescript_dprint_options = '-c dprint.json'
+
+  AssertFixer
+  \ {
+  \   'read_temporary_file': 1,
+  \   'command':
+  \     ale#Escape('dprint')
+  \     . ' ' . ale#Escape('-c dprint.json')
+  \     . ' %t',
+  \ }
```

Comment:
Same as previous test make sure the dictionary matches exactly what the fale#fixers#dprint#Fix() method returns:

```
    \ {
    \   'command': ale#Escape('dprint') . ' fmt -c dprint.json --stdin %s',
    \ }
```

Reply:
Thanks. On it. Sorry to take some time, I saw the failures but haven't been able to address them yet. 💪